tests/h/tasks/indexer_test.py

```diff
@@ -9,119 +9,29 @@
 from h.tasks import indexer
 
 
-class FakeSettingsService:
-    def __init__(self):
-        self._data = {}
+class TestSearchIndexServicesWrapperTasks:
+    """Tests for tasks that just wrap SearchIndexServices functions."""
```

Comment:
You can't see this very easily in the diff, but I pulled the two tests which just delegate off to `SearchIndexServices` here. My fond hope is that more and more of them end-up like this.